web/html/src/manager/admin/access-control/access-group-list.tsx

```diff
@@ -0,0 +1,152 @@
+import * as React from "react";
+import { useEffect, useState } from "react";
+
+import { Button, LinkButton } from "components/buttons";
+import { DeleteDialog } from "components/dialog/DeleteDialog";
+import { ModalButton } from "components/dialog/ModalButton";
+import { Messages, MessageType, Utils as MessagesUtils } from "components/messages/messages";
+import { TopPanel } from "components/panels/TopPanel";
+import { Column } from "components/table/Column";
+import { SearchField } from "components/table/SearchField";
+import { Table } from "components/table/Table";
+
+import { Utils } from "utils/functions";
+import Network from "utils/network";
+
+type AccessGroupListItem = {
+  id: number;
+  name: string;
+  description: string;
+  orgName: string;
+  numUsers: number;
+  numPermissions: number;
+};
+
+const AccessGroupList = (props) => {
+  const [accessGroups, setAccessGroups] = useState<AccessGroupListItem[]>([]);
+  const [messages, setMessages] = useState<MessageType[]>([]);
+  const [toDelete, setToDelete] = useState<AccessGroupListItem>();
+
+  const searchData = (item, criteria) => {
+    if (!criteria) {
+      return true;
+    }
+    const lowerCaseCriteria = criteria.trim().toLowerCase();
+    return (
+      item.name?.toLowerCase().includes(lowerCaseCriteria) ||
+      item.description?.toLowerCase().includes(lowerCaseCriteria)
+    );
+  };
+
+  useEffect(() => {
+    getRoles();
+  }, []);
+  const getRoles = () => {
+    const endpoint = "/rhn/manager/api/admin/access-control/access-group/list_custom";
+    Network.get(endpoint)
+      .then((groups) => {
+        setAccessGroups(groups);
+      })
+      .catch(() => {
+        setMessages(
+          messages.concat(MessagesUtils.error(t("An unexpected error occurred while fetching access groups.")))
+        );
+      });
+  };
+
+  const onDelete = (item) => {
+    return Network.del("/rhn/manager/api/admin/access-control/access-group/delete/" + item.id)
+      .then((_) => {
+        setMessages(MessagesUtils.info("Access Group '" + item.name + "' has been deleted."));
+        setAccessGroups(accessGroups.filter((g) => g.id !== item.id));
+      })
+      .catch((error) => setMessages(Network.responseErrorMessage(error)));
+  };
+
+  const actionButtons = (item: AccessGroupListItem) => {
+    if (item.orgName === "-") {
+      return (
+        <div className="btn-group">
+          <Button className="btn-default btn-sm" icon="fa-user" />
```

Comment:
This button has no function. I think we don't need it at all. I don't even remember why this was here in the first place (also below in the `else` block).